src/frontend/src/lib/components/registry/teams/Tag.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+  import type { Team } from '$lib/types/team';
+  export let team: Team;
+  import { colors } from '$lib/stores/colors';
+
+  $: color = colors[team.color - 1].value;
```

Comment:
Through testing the branch on my machine this line is throwing errors on any page the Tag component is used and the page won't load, worth looking into:
![image](https://github.com/MarioKartCentral/MarioKartCentral/assets/46006425/5d46ec47-3a27-4556-9619-143c68830ead)

Reply:
This error happens when the team color is invalid, the value has to be between 1 and 40 and it has the NOT NULL constraint
I can still provide a default case if needed when the team color is invalid though